devlib/module/thermal.py

```diff
@@ -68,7 +70,25 @@ def set_enabled(self, enabled=True):
     def get_temperature(self):
         """Returns the temperature of the thermal zone"""
         temp_file = self.target.path.join(self.path, 'temp')
```

Comment:
`temp_file` is strongly associated with "temporary file" concept. Unless the rest of the module uses that convention, I'd suggest use something else like `temperature_file` to avoid any confusion